setup.py

```diff
@@ -14,25 +14,21 @@
 
 from os.path import dirname, join
 import plyer
+from setuptools import setup
 import io
 
 EXTRA_OPTIONS = {}
```

Comment:
I guess we do not need this one anymore 😀

And, since we now only use `setuptools`, how about moving the whole extra options into `setup(` ?